marklogic-data-hub-central/ui/src/components/entities/merging/add-merge-rule/add-merge-rule-dialog.tsx

```diff
@@ -418,16 +432,17 @@ const AddMergeRuleDialog: React.FC<Props> = (props) => {
                                 validateStatus={(strategyValue || !strategyValueTouched) ? '' : 'error'}
                                 help={(strategyValue || !strategyValueTouched) ? '' : 'Strategy Name is required'}
                             >
-                                <MLInput
+                                <MLSelect
                                     id="strategyName"
-                                    placeholder="Enter strategy name"
+                                    placeholder="Select strategy name"
                                     size="default"
                                     value={strategyValue}
```

Comment:
This placeholder will never be visible for this field because the default value here is empty or null. It should be **undefined** instead of empty string.